internal/engine/build_and_deployer.go

```diff
@@ -54,7 +54,7 @@ func (composite *CompositeBuildAndDeployer) BuildAndDeploy(ctx context.Context,
 			logger.Get(ctx).Debugf("(expected error) falling back to next build and deploy method "+
 				"after error: %v", err)
 		} else {
-			logger.Get(ctx).Verbosef("falling back to next build and deploy method "+
+			logger.Get(ctx).Infof("falling back to next build and deploy method "+
```

Comment:
Was this intended to be left at Info? I think we had general agreement this was more distracting than useful.

Reply:
There are now two cases:
a) expected error, at DebugF
b) unexpected error, which I think is worth upping to Info

But don't feel super strongly.